source/concepts/faq.md

```diff
@@ -60,6 +60,36 @@ There are downsides to relying on [experimental features](https://nixos.org/manu
 - The [Nix documentation team](https://nixos.org/community/teams/documentation.html) focuses on improving documentation and learning materials for stable features and common principles.
   When using flakes, you will have to rely more heavily on user-to-user support, third-party documentation, and the source code.
 
+## Which channel branch should I use?
+
+Nixpkgs and NixOS have both stable and rolling releases.
+
+### Stable
+
+- On Linux (including NixOS and WSL), use [`nixos-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixos-).
+
+  These branches only contain commits that passed the NixOS test suite.
+- On any other platform, use [`nixpkgs-*`](https://github.com/NixOS/nixpkgs/branches/all?query=nixpkgs-).
+
+  These branches contain commits that passed the test suites specific to other platforms.
```

Comment:
I really don't think this is right, as [mentioned before](https://github.com/NixOS/nix.dev/pull/744#discussion_r1380373641). The only stable `nixpkgs-*` channels are the Darwin ones like `nixpkgs-23.05-darwin`, but we cannot assume that all non-Linux platforms must be Darwin, or that the darwin jobsets will run tests on those platforms. This is just confusing like this.

Reply:
Opened https://github.com/NixOS/nix.dev/pull/791 for this